app/src/main/java/com/douglasbruce/tasky/core/domain/mapper/EventMapper.kt

```diff
@@ -35,18 +36,23 @@ fun EventEntity.toEvent(): AgendaItem.Event {
         eventDescription = this.description,
         from = ZonedDateTime.ofInstant(Instant.ofEpochMilli(this.from), ZoneId.systemDefault()),
         to = ZonedDateTime.ofInstant(Instant.ofEpochMilli(this.to), ZoneId.systemDefault()),
-        remindAtTime = ZonedDateTime.ofInstant(Instant.ofEpochMilli(this.remindAt), ZoneId.systemDefault()),
+        remindAtTime = ZonedDateTime.ofInstant(
+            Instant.ofEpochMilli(this.remindAt),
+            ZoneId.systemDefault()
```

Comment:
I'm not 100% sure right now, but does this function convert the millis into the local user's time zone or does it interpret it in the local user's time zone? I feel like the latter is true - in that case you'd need to pass `ZoneId.of("UTC")` and convert it to the local user's timezone

Reply:
I may be wrong, but I had the belief that this would convert the long value, which is UTC, to the local user's timezone